downstream/modules/aap-hardening/con-external-credential-vault.adoc

```diff
@@ -7,8 +7,11 @@
 
 [role="_abstract"]
 
-Secrets management is an essential component of maintaining a secure automation platform. We recommend the following secrets management practices:
+Secrets management is an essential component of maintaining a secure automation platform. 
+We recommend the following secrets management practices:
```

Comment:
Consider changing "We recommend" to "Consider...", similar to my previous comment. Unless our team has decided this is ok? (Personally, I don't mind using "recommend".)